### PR TITLE
Added hover animations to post icons

### PR DESCRIPTION
**Pull request by AdityaChaudhary0005**
Preview of hover animations on post icons: 

Hover PR

### CODE REVIEW

**Comment by teambrixmusic** [comment]
So cool!

**Comment by tijno** [comment]
@AdityaChaudhary0005 nice! you might want to check with sourdough on bitclout or cloutedones discord - i know she had suggestion for making the icon animation better for accessibility on mobile devices. May be an opportunity to consider that.

**Comment by AdityaChaudhary0005** [comment]
Thanks @teambrixmusic and @tijno !

yeah, i will consider contacting sourdough!

File: src/styles.scss [hunk 1]
```diff
@@ -5,6 +5,8 @@ $success: #32ED4B;
 $info: #689eff;
 $feed-border-radius: 12px;
 
+
+
```
Comment (lazynina) [inline src/styles.scss:9]:
```suggestion
```

**Review by lazynina — CHANGES_REQUESTED** [review]
this looks good to me!  @tijno, we can consider improvements to the icon animation for mobile in a separate PR.

**Review by lazynina — APPROVED** [review]

**Pull request merged**